### DISCUSSION TITLE
Projeto 17 - Certificado Python

### DISCUSSION THREAD

**Discussion by LuizFernando-TC** [codaqui/institucional#54] [Certificados]
Sugestão para o projeto 17 do certificado de Python:
1. Seu programa deverá exibir o dia da semana que foi 07/09/2022
2. Seu programa deverá exibir a difereça de dias entre 11/11/1918 e 01/09/1939

**Comment by endersonmenezes** [comment]
Bacana @LuizFernando-TC! 
O que acha de inserir algum exercício que trabalha com conversão de datas, e conversão entre tipos de variável?
Um tipo string se tornar data, e um tipo data se tornar string? Acho que mais alguns exercícios podem ficar mais interessantes ai =]

**Comment by endersonmenezes** [comment]
Issue: https://github.com/codaqui/institucional/issues/55
PullRequest: https://github.com/codaqui/certificado-python/pull/9

**Comment by LuizFernando-TC** [comment]
Acho bom, Enderson.
Porém, você acredita que essa trilha de estudo fornece material suficiente para isso?

**Reply by endersonmenezes** [reply]
Acho que quem compreender de uma forma mais profunda a trilha iria conseguir, será então que a gente coloca como um DESAFIO? E alguma mensagem do tipo: "Não se preocupe, os exercícios abaixo não desclassificarão o seu certificado, mas se você conseguir resolve-los terá uma nota extra."

**Reply by LuizFernando-TC** [reply]
Então podemos adicionar mais exercícios e deixar no certificado normal. No final do certificado poderia ter um desafio que abrange vários conteúdos. O que acha?. Deixar o desafio como você disse.

**Reply by LuizFernando-TC** [reply]
Acredito que seja necessário uma revisão na trilha também, mais para frente. Pode ser feita uma trilha de nível 2 de Python.

**Reply by endersonmenezes** [reply]
Poderia ser a mesma trilha, pra manter 1 trilha por linguagem. Explicar a teoria, e dentro dos conteúdos separar por nível? Você acha que tudo hoje na trilha está básico? Podemos pensar em 3 níveis, iniciante, intermediário e avançado.

**Reply by LuizFernando-TC** [reply]
Bom, eu não sei exatamente como é o nível intermediário e avançado no python, já que eu estou no básico. Acredito que o nível intermediário e avançado abordaria assuntos mais específicos. A trilha apresenta o necessário para programar em python. Seria bom algumas pessoas analisarem a trilha também. Juntar conteúdos de python e analisar cursos de python ajudariam a ver como está a trilha.

**Reply by endersonmenezes** [reply]
Então eu acho que poderiamos ter a nossa trilha de programação, independente de linguagem abordando os conceitos do básico ao avançado igual a gente faz. E ter Desafio/Projetos que testem a pessoa para ela entender o que seria um avançado e o que seria um básico. O que a gente ensina é uma base geral. =]
